tests/parsing/swift/statements.swift

```diff
@@ -48,22 +48,22 @@ var y { }
 var x {
   get { return 2 }
   _modify { }
-  set { return () }
+  set { return 2 }
```

Comment:
why did you need to change this test file?

Reply:
I just fixed this while I was here, basically, because this was breaking some of Martin's tests and I think he xfailed it. This was hand-in-hand with the missing nodes, because this is an invalid Swift program, and this causes us to suddenly fail.